Dockerfile

```diff
@@ -51,28 +51,49 @@ ARG NVIDIA_DALI_BUILD_FLAVOR
 ARG GIT_SHA
 ARG DALI_TIMESTAMP
 
+# Optional build arguments
+
+ARG BUILD_TEST=ON
+ARG BUILD_BENCHMARK=ON
+# ARG BUILD_NVTX=ON
```

Comment:
Please add this one (may be off by default) and propagate it to BUILD_NVTX.

Reply:
Done.